release-notes/rn-axon-framework/rn-af-major-releases.md

```diff
@@ -2,6 +2,12 @@
 
 All the enhancements and features which have been introduced to our major releases of the Axon Framework are noted here.
 
+## Release 4.7
```

Comment:
Great start of the notes :muscle:. We'll definitely need to include more, or refer to some (migration) blog on the potential harshness of the Javax/Jakarta adjustments that followed the change to Spring Boot 3/Spring Framework 6.